hack/generator/pkg/astmodel/types.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+import "fmt"
+
+// Types is the set of all types being generated
```

Comment:
minor: This comment supposes a particular usage.

```suggestion
// Types is a map of TypeName to TypeDefinition, representing a set of types.
```